### PR TITLE
Add resources page

### LINKED ISSUE #81: Add Detroit Resource center section
Blocked by HRD collecting resources

- [ ] Work with Tom to determine if it should look like the Bloom resource section
- [ ] Work with HRD to determine what links and resources we need
- [ ] Link to HRC intake form
- [ ] Get Help section on homepage AND listing detail page

### PR DESCRIPTION
**Pull request by GabeBenjamin**
## Issue

- Closes #81

## Description

Adds a resources page with the general outline and strings that we
currently have based on:
https://docs.google.com/document/d/16XULtYdy1XJ9aBBty-BlQe8nYbQlJ-s1w20-t8CrcJU/edit

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Prototype/POC (not to merge)
- [ ] This change is a refactor/addresses technical debt
- [ ] This change requires a documentation update
- [ ] This change requires a SQL Script

### Mobile
image

### Desktop
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have updated the changelog to include a description of my changes
- [ ] I have run `yarn generate:client` if I made backend changes
- [ ] I have exported any new pieces in ui-components

### CODE REVIEW

File: sites/public/pages/resources.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+const Feedback = () => {
+  const pageTitle = <>{t("pageTitle.resources")}</>
+
+  return (
+    <Layout>
+      <PageHeader title={pageTitle} inverse />
+      <div className="px-5 max-w-5xl md:m-auto">
+        <p className="mt-8">{t("resources.body1")}</p>
+        <div className="mt-8">
```
Comment (plunkettgoogle) [inline sites/public/pages/resources.tsx:12]: Is there value adding these sections with no accompanying text or links? Maybe we could add them later when they contain more information?

File: sites/public/pages/resources.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+          <h2>{t("resources.evictionAssistance")}</h2>
+        </div>
+        <div className="mt-8">
+          <h2>{t("resources.detroitHousingNetwork")}</h2>
+          <p>
+            {t("resources.detroitHousingNetworkBody")}{" "}
+            {/* It isn't ideal to make the url separate from the translated text but polyglot
```
Comment (plunkettgoogle) [inline sites/public/pages/resources.tsx:19]: Could you use interpolation here? https://github.com/airbnb/polyglot.js#interpolation

Comment (GabeBenjamin) [reply sites/public/pages/resources.tsx:19]: I tried that, but it was resulting in printing out [object Object] so I think they only allow strings for interpolation and not anything else

Comment (plunkettgoogle) [reply sites/public/pages/resources.tsx:19]: Wait, so substituting the string `"<a href="https://detroithousingnetwork.org">https://detroithousingnetwork.org</a>"` doesn't work? Like literally putting in the string representation of the html link?

Comment (GabeBenjamin) [reply sites/public/pages/resources.tsx:19]: Putting in that as a string will result in it outputting the literal string `"<a href="https://detroithousingnetwork.org">https://detroithousingnetwork.org</a>"` but we want it to be the semantic html tag which doesn't work.

Looks like there's an open issue on polyglot to add this: https://github.com/airbnb/polyglot.js/issues/93

Sounds like we could add wrapper interpolations and manually call `dangerouslySetInnerHTML` but I'm not sure that's worth it for this. It's probably better to wait until polyglot officially supports it.

Comment (plunkettgoogle) [reply sites/public/pages/resources.tsx:19]: Ah, okok, bummer. :(

**Review by plunkettgoogle — APPROVED** [review]

File: sites/public/pages/resources.tsx (hunk 1 above)
Comment (GabeBenjamin) [reply sites/public/pages/resources.tsx:12]: This is what Safiya requested for now so I think it's useful to:

1. Get at least the headings in for translations
2. Act as an outline/reminder for what information we still need to gather for this page

**Pull request merged**